rt_entt_codegen/core/execute_systems.cc

```diff
@@ -1,14 +1,44 @@
 #include "core.hh"
 
+#include "rt_entt_codegen/shared/parallel.hh"
 #include "ecsact/lang-support/lang-cc.hh"
 #include "rt_entt_codegen/shared/util.hh"
 #include "ecsact/cpp_codegen_plugin_util.hh"
 
+#include <iostream>
+
 constexpr auto METHOD_BODY_TOP = R"(
-auto& reg = ecsact::entt::get_registry(registry_id);
-auto actions = ecsact::entt::actions_map{};
+auto& registry = ecsact::entt::get_registry(registry_id);
+auto actions_map = ecsact::entt::actions_map{};
 )";
 
+auto ecsact::rt_entt_codegen::core::print_parallel_system_execute(
+	codegen_plugin_context&    ctx,
+	const ecsact_entt_details& details
+) -> void {
+	using ecsact::cc_lang_support::cpp_identifier;
+	using ecsact::cpp_codegen_plugin_util::block;
+	using ecsact::rt_entt_codegen::util::method_printer;
+
+	auto printer = //
+		method_printer{ctx, "ecsact_execute_parallel_cluster"} //
+			.parameter("::entt::registry&", "registry")
+			.parameter("ecsact_system_execution_context*", "parent_context")
+			.parameter("auto", "system_arr")
+			.return_type("void");
+
+	ctx.write(
+		"std::for_each(std::execution::par_unseq\n, system_arr.begin()\n, "
+		"system_arr.end()\n, "
+	);
+	block(ctx, "[&](auto pair)", [&]() {
+		ctx.write("auto fn_ptr = pair.first;\n");
+		ctx.write("auto actions_map = pair.second;\n");
+		ctx.write("fn_ptr(registry, nullptr, actions_map);");
```

Comment:
just to avoid a copy lets throw actions map right into the fn as a param (or add `&`)
```suggestion
		ctx.write("fn_ptr(registry, nullptr, pair.second);");g
```

Reply:
I'll make it a reference for readability